### DISCUSSION TITLE
Haxe Migration

### DISCUSSION THREAD

**Discussion by drunkcat** [prominic/Moonshine-IDE#797] [General]
Hello!
Just a general question - what's it's all about with the "Haxe Migration" thing? Is the whole project migrating to HaXe? :smile:
Greg.

**Comment by piotrzarzycki21** [comment]
Hello @drunkcat ,

We have decided to move Moonshine source code to Haxe. :) We are doing it slowly and incrementally. If you are using Nightly build of Moonshine some views are already written in Haxe - they are launched in special wrapper which allows run them in Flex.

**Comment by joshtynjala** [comment]
Here's a blog post about the new Haxe and Feathers UI text editor, if anyone is interested:

https://feathersui.com/blog/2021/11/09/moonshine-ide-feathers-ui-code-text-editor/
